Realm.podspec

```diff
@@ -38,5 +38,6 @@ Pod::Spec.new do |s|
 
   s.subspec 'Headers' do |s|
     s.source_files          = 'include/**/*.h'
+    s.private_header_files  = '**/*.hpp', '**/*_Private.h', '**/*_Dynamic.h', 'include/tightdb/**/*.h'
```

Comment:
Sure, it's just Ruby. But that will decrease readability slightly, though I doubt that a lot external developers will take a look into the actual podspec.